### PR TITLE
Bs template

### PR DESCRIPTION
**Pull request by Nightwarp3**
Merging the bsTemplate branch to the master branch. All functionality remains, with a more aesthetically pleasing look.

### CODE REVIEW

**Comment by inxfidxel** [comment]
nice!!!

File: NightwarpComputers/Views/Routes/Submitted.cshtml [hunk 1]
```diff
@@ -4,7 +4,7 @@
             <h2 class="text-success">Order has been submitted successfully!</h2>
         </div>
         <div class="row justify-content-center">
-            <p>Order ID: <b><i>{{orderId}}</i></b> has been receieved. You will be receiving a confirmation email shortly at: <b>{{email}}</b> from "NightwarpComputers@outlook.com.</p>
```
Comment (inxfidxel) [inline NightwarpComputers/Views/Routes/Submitted.cshtml]: Are you not using angularjs anymore?

Comment (Nightwarp3) [reply NightwarpComputers/Views/Routes/Submitted.cshtml]: I am, I just found it easiest on this page to use the parameters I passed to my ASP RouteController in the ViewBag and access the values via Razor in the html. Although I do have the parameters in the url on this page, so I could grab them from my AngularJS script.

Comment (inxfidxel) [reply NightwarpComputers/Views/Routes/Submitted.cshtml]: okay cool, I was just wondering

**Pull request merged**